ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/lexer/LexerFileProcessor.java

```diff
@@ -0,0 +1,27 @@
+package edu.usb.argos.ASTProcessor.lexer;
+
+import edu.usb.argos.ASTProcessor.antlr.JavaLexer;
+import edu.usb.argos.ASTProcessor.lexer.errorHandler.ArgosErrorListener;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class LexerFileProcessor {
+    public CommonTokenStream getTokensFromFile(String filePath) throws IOException {
+        Path path = Paths.get(filePath);
+        String content = new String(Files.readAllBytes(path));
+
+        JavaLexer lexer = new JavaLexer(CharStreams.fromString(content));
+
+        ArgosErrorListener errorListener = new ArgosErrorListener();
```

Comment:
The `errorListener` should be a private field of the class and initialized in the constructor

Reply:
Done ✅